src/fides/cli/commands/util.py

```diff
@@ -27,6 +30,28 @@
 from fides.core.utils import echo_green
 
 
+def create_and_update_config_file(
```

Comment:
This function is only used for these CLI commands, but maybe it should live elsewhere?